### DISCUSSION TITLE
Logo 设计

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [LinuxSuRen/api-testing#169] [General]
下面是我设计的一个 Logo 草图，仅仅是为了项目有个代表性的图片，欢迎感兴趣的朋友们一起设计：

![api-testing](https://github.com/LinuxSuRen/api-testing/assets/1450685/9645b064-7aaf-4de9-94a4-df4c41a11fac)

**Comment by yuluo-yx** [comment]
草图：

![logo-1](https://github.com/LinuxSuRen/api-testing/assets/77964041/6a9c2571-ff8b-4189-93c0-39cfdfab619d)

地球表示需要测试项目实体；
连线表示测试，从一个点到一个点，验证一个可行性；
蓝色给人稳重，稳定&成熟的感觉；
最后加入 API Testing 字样。 🚀

这参考了github 首页的地球形状，如果可行性较高，我们可以使用 svg 将此图形给绘制出来，更形象生动一些。

**Reply by wjsvec** [reply]
很好看呀☺️

**Reply by LinuxSuRen** [reply]
这个是手绘的吗，我们得确保版权上没有问题。

有一个小疑问是，从上面的图看不到和`测试`或`接口`有明显的关联，而且地球有点过于通用，可以考虑适当地进行变形下。

**Reply by yuluo-yx** [reply]
目前这只是一个参考的 logo 方案，（因为不太擅长 logo 设计方面，仅仅只是在草图设想层面。
需要一个比较专业的人去搞，加一些个性化内容或者其他元素

**Comment by KariHall619** [comment]
hey there!
[logo.zip](https://github.com/user-attachments/files/20348651/logo.zip)

**Reply by KariHall619** [reply]
![last-version](https://github.com/user-attachments/assets/8300e838-d7f2-4963-a125-92d49885b2ee)

**Reply by yuluo-yx** [reply]
可以哇，我觉得很不错 👍

**Reply by KariHall619** [reply]
> 可以哇，我觉得很不错 👍

感谢认可😉
